storagetransfer/nearline_request.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env python
```

Comment:
Please remove these lines

Reply:
This (and others) are CLI-enabled commands (runnable via `./nearline_request.py`): https://github.com/GoogleCloudPlatform/python-docs-samples/blob/main/AUTHORING_GUIDE.md#shebang

Reply:
Ah, other shebang's aren't consistent for example: `get_transfer_job_with_retries_apiary.py` uses `#!/usr/bin/env`. Could you address those (doesn't need to be in the same PR)

Reply:
Good catch - updated